### DISCUSSION TITLE
Dracula UI status/updates?

### DISCUSSION THREAD

**Discussion by bigpick** [dracula/dracula-theme#511] [Q&A] [answered]
Ever since the [Dracula UI](https://draculatheme.com/ui) button got added to the Dracula site, I've been waiting to see the awesome work.

We've gotten a few teasers in the [Week 1](https://draculatheme.com/blog/dracula-ui-week-1) and [Week 3](https://draculatheme.com/blog/dracula-ui-week-3) updates (which look so nice!), but that was back in September 10/24 2020, respectively.

I know there's the mailing list (which I've been on:)) but other than that, it seems to have gone quiet... 

1. I hope all is well with the three of you.
2. Is there plans on continuing the Dracula UI work? Or maybe, it has been, and I just missed the updates?

I've been looking at updating at least my personal site, and was holding off for potentially an officially supported Dracula UI toolbox to work with, but at this point am curious what to expect. Would be awesome to have that.

As a side note, the content that was posted in the two week updates look really nice. [My Github README](https://github.com/bigpick) now uses those exact gradients ❤️ 🚀 : 

![dracula-git](https://user-images.githubusercontent.com/9803299/107065481-d4f72500-67aa-11eb-962b-0d020455bd0a.png)

**Comment by zenorocha** [comment]
Hey @bigpick!

Thank you so much for reaching out. I really appreciate the fact that you're looking forward to DraculaUI and that you already used some colors on your README (looks great btw).

Last year many things changed in our personal/work lives for both me and @nettofarah. Because of that, we had to put a pause on this project. However, that doesn't mean we have given up on it. On the contrary, we're still very excited about this.

I wish I could give you a proper timeline, but all I can say now is that I'm confident that we'll release something this year and you can expect a high-quality package of HTML/CSS/JS components.

**Reply by bigpick** [reply]
And totally understandably so. I figured as much, but was always just a presumption. Glad to hear that things are still planned, looking forward to it.

Cheers!

**Comment by zenorocha** [comment, marked as answer]
And it's live! https://draculatheme.com/ui
